worlds/v6/__init__.py

```diff
@@ -61,3 +61,27 @@ def fill_slot_data(self):
             "DeathLink": self.world.DeathLink[self.player].value,
             "DeathLink_Amnesty": self.world.DeathLinkAmnesty[self.player].value
         }
+
+    def generate_output(self, output_directory: str):
+        if self.world.players != 1:
+            return
+        try:
```

Comment:
try -> except anything -> raise 
is the same as just not having it, any uncaught exception will be raised.

Reply:
Removed it